### PR TITLE
Fix canjs6 update errors

### PR DESCRIPTION
**Pull request by cherifGsoul**
Fixes:
- Use `canReflect`
- Type errors

Fixes https://github.com/canjs/devtools/issues/85

### CODE REVIEW

File: bindings-graph/bindings-graph.mjs [hunk 1]
```diff
@@ -47,7 +47,7 @@ class CanjsDevtoolsBindingsGraph extends StacheElement {
 
 			selectedObj: String,
 			selectedKey: String,
-			bindingsError: String
+			bindingsError: type.convert(String)
```
Comment (phillipskevin) [inline bindings-graph/bindings-graph.mjs]: what other type was being passed for this?

File: panel/panel.mjs [hunk 2]
```diff
@@ -237,7 +242,7 @@ export default class CanjsDevtoolsPanel extends StacheElement {
 						vm.breakpointsError = detail;
 						break;
 					case "success": {
-						vm.componentTree.updateDeep(detail.tree || []);
+						Reflect.updateDeep(vm.componentTree, detail.tree || []);
```
Comment (phillipskevin) [inline panel/panel.mjs]: There must not be tests for this. Can we add some?

Comment (cherifGsoul) [reply panel/panel.mjs]: Will do!

File: bindings-graph/bindings-graph.mjs (hunk 1 above)
Comment (cherifGsoul) [reply bindings-graph/bindings-graph.mjs]: It could be `null` or `undefined`

Comment (phillipskevin) [reply bindings-graph/bindings-graph.mjs]: Then it should be `type.maybe(String)`

Comment (cherifGsoul) [reply bindings-graph/bindings-graph.mjs]: I'll push the update of this property with bindings graph fix, thanks!

**Pull request merged**